### PR TITLE
[hailctl] Fix labeling to support arbitrary inputs

### PR DESCRIPTION
**Pull request by tpoterba**

### CODE REVIEW

**Comment by tpoterba** [comment]
it was totally broken before, actually.

File: hail/python/hailctl/dataproc/start.py [hunk 1]
```diff
@@ -173,8 +173,11 @@ def main(args, pass_through_args):
     if args.bucket:
         conf.flags['bucket'] = args.bucket
 
-    label = safe_call('gcloud', 'config', 'get-value', 'account')
-    conf.flags['labels'] = 'creator=' + label.replace('.', '-dot-').replace('@', '-at-')  # no special chars
+    try:
+        label = sp.check_output('gcloud config get-value account', shell=True)
```
Comment (chrisvittal) [inline hail/python/hailctl/dataproc/start.py]: Why are we invoking a shell here?

Comment (tpoterba) [reply hail/python/hailctl/dataproc/start.py]: the alternative doesn't work. The way gcloud picks up credentials is totally mystifying.

Comment (tpoterba) [reply hail/python/hailctl/dataproc/start.py]: oh wait, yes it does

Comment (tpoterba) [reply hail/python/hailctl/dataproc/start.py]: will fix

**Pull request merged**